GCDS.NetTemplate/Components/ILink.cs

```diff
@@ -10,6 +10,11 @@ public interface ILink
 
     public static class ListLinkExtensions
```

Comment:
Validate the value of the ILink interface, this could potentially just target the `Link` class.

Reply:
To be handled in the future.